.github/workflows/actions.yml

```
@@ -0,0 +1,47 @@
+name: Continuous Integration
+on:
+  pull_request:
+    branches:
+      - master
+  push:
+    branches:
+      - master
+jobs:
+  specs:
+    runs-on: zendesk-stable
+    strategy:
+      fail-fast: false
+      matrix:
+        ruby:
+          - 2.6.6 # Current Classic
+          - 2.7.1
+        gemfile:
+          - gemfiles/rails4.2.gemfile
+          - gemfiles/rails5.0.gemfile
+          - gemfiles/rails5.1.gemfile
+          - gemfiles/rails5.2.gemfile
+        exclude:
+          - {ruby: 2.7.1, gemfile: 'gemfiles/rails4.2.gemfile'} # NoMethodError: undefined method `new' for BigDecimal:Class
+    steps:
+      - uses: zendesk/checkout@v2
+      - uses: zendesk/setup-ruby@v1.48.0
```

Why the version pin? It should be @v1 unless there's a specific reason.

```suggestion
      - uses: zendesk/setup-ruby@v1
```

---

copied from another project 🙈